src/auth/oidc.ts

```diff
@@ -162,7 +162,17 @@ async function callIdpTokenEndpoint (usp: URLSearchParams, req: express.Request,
 
         // After login redirect to the dashboard, but otherwise return a bearer token
         if (isLogin) {
-            return res.redirect(`${new URL(`${RuntimeConfig.dashboardAddress}`, ServerConfig.serverAddress).href}?${new URLSearchParams(`oidcuser=${username}`).toString()}`);
+            const loginUsp = new URLSearchParams();
+            loginUsp.set('oidcuser',`${username}`);
+            if (req.cookies['redirectParams']) {
+                loginUsp.set('redirectParams', req.cookies['redirectParams']);
```

Comment:
why don't we just use a session cookie here? where does max age come from in this regard? wouldn't a session cookie be cleaner, as we wouldn't leave it hanging around after using it

Reply:
We could perhaps use a session cookie here - I'm just reusing the same code as with OIDC verifier here for simplicity of implementation (which is also where the maxAge was taken from, though if trying to clear the cookie it should probably be set to 0, assuming that's a valid value).

Not sure if maxAge there of 0 would be better than a session cookie?  I don't see session cookies (assuming [this](https://expressjs.com/en/resources/middleware/session.html) is what you meant) in use elsewhere in that codebase, so guessing it might actually require more work on the server side as the Express documentation says the following about it:

> Warning The default server-side session storage, MemoryStore, is purposely not designed for a production environment. It will leak memory under most conditions, does not scale past a single process, and is meant for debugging and developing.